variables.tf

```diff
@@ -109,3 +109,45 @@ variable "enable_diagnostic_setting" {
   description = "Enable diagnostic setting. var.analytics_workspace_id must be provided"
   default     = false
 }
+
+variable "keyvault_key_permissions" {
```

Comment:
Change variable name to "key_permissions" and for other related variables.
Update readme with all created variables